Automattic/wp-calypso
# Gutenboarding: update design preview screenshots

## Changes proposed in this Pull Request

Hey, thanks for dropping by. This PR isn't the most exciting code change, but we hope you have fun reviewing nonetheless. 

There's no code to pick apart. Put on some music. Maybe some chill trance, or heavy metal, or whatever you like.

So we pushed a few changes to the starter site copy in D47680-code. This PR updates the corresponding design screenshots to reflect these changes.

<img width="717" alt="Screen Shot 2020-08-13 at 12 40 40 pm" src="https://user-images.githubusercontent.com/6458278/90088410-368d8800-dd62-11ea-9fca-9c386ef656dc.png">

## Testing instructions

Fire up this branch and head over to `/new` 

Compare the following screenshots with their previews 

```
bowen
cassel
coutoire
easley
edison
leven
reynolds
```

The copy should be the same and, more importantly, we should enjoy that which is good.
